packages/web/src/pages/profile-page/components/desktop/RelatedArtists.tsx

```diff
@@ -60,6 +62,11 @@ export const RelatedArtists = () => {
       <ProfilePictureListTile
         onClick={handleClick}
         users={relatedArtists as User[]}
+        totalUserCount={
```

Comment:
why do we need another const, can we ust use MAX_PROILE_RELATED_ARTISTS?

Reply:
this is just how it was written before. think you're right though